### PR TITLE
prepare for `0.12.0`

### PR DESCRIPTION
**Pull request by BurnzZ**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,16 @@
 Changelog
 =========
 
+0.12.0 (2023-10-24)
+===================
+
+* Added :meth:`~zyte_common_items.base.ProbabilityMixin.get_probability` helper
+  method in page classes (e.g. :class:`~zyte_common_items.Product`,
```
Comment (kmike) [inline CHANGELOG.rst:9]: Isn't it also added for the item classes, not just for page classes?

Comment (BurnzZ) [reply CHANGELOG.rst]: Ohh right, I got mixed up by the previous terminologies that were use in the previous release. I just realized it was _"page classes **for** BusinessPlace, JobPosting, ..."_ https://github.com/zytedata/zyte-common-items/pull/66/commits/cf0abd43de8cd9ec4730f76c937d506c4cd93ec0

**Comment by kmike** [comment]
@BurnzZ sorry, I merged it, but I should have let you to do the release after the approval - could you please ensure the release is correct (dates, tags, etc)?

**Comment by BurnzZ** [comment]
No worries, updated the date in https://github.com/zytedata/zyte-common-items/commit/b2f9b6dec4a8251fd4982d1f10ddbcda34e413e4

**Pull request merged**